### DISCUSSION TITLE
Specify annoPath in addArchAnnotations

### DISCUSSION THREAD

**Discussion by ChloeHJ** [GreenleafLab/ArchR#1897] [Feature Requests]
Hi, since I am running archR via server, I often don't have writing permission to the R packages and addArchAnnotations that require downloading the Anno files to R/ directories fail. Is it possible to add a parameter in the addArchAnnotations function to specify the annoPath? 

Many thanks!

**Comment by rcorces** [comment]
I've added this feature on a new branch called `dev_annoPath` via https://github.com/GreenleafLab/ArchR/commit/c9742ad5793478d1d10caa09b8955966a1d5d4cf. I have not tested the additions and would appreciate if you could install this branch and report back if it works.

```R
devtools::install_github("GreenleafLab/ArchR", ref="dev_annoPath", repos = BiocManager::repositories(), upgrade = "never")
#to unload a package and reload
detach("package:ArchR", unload=TRUE)
library(ArchR)
```
